jquery.kinetic.js

```diff
@@ -226,6 +224,13 @@
         if (self._useTarget(e.target, e) && self.elementFocused){
           return false;
         }
+      },
+      // prevent selection when dragging
+      selectStart: function (e){
+        if ($.isFunction(self.settings.selectStart)){
```

Comment:
After thought on this, did you see my comment yesterday? Basically there's no point allowing drag and selecting at the same time so this would need to prevent drag as well ... 

therefor it makes sense to me to just allow selection when `!self._useTarget(e.target, e)` so either we use the target for dragging and prevent selection or we don't use it for dragging and allow click, selection etc etc